pubspec.yaml

```diff
@@ -1,7 +1,9 @@
 name: cider
-version: 0.0.4
+version: 0.0.4-allow-setting-version-explicitly
```

Comment:
`0.0.5` should be OK

Reply:
My idea is that you fix things yourself, and then bump version c: